bullet-featherstone/src/JointFeatures.cc

```diff
@@ -80,16 +80,42 @@ double JointFeatures::GetJointAcceleration(
 double JointFeatures::GetJointForce(
     const Identity &_id, const std::size_t _dof) const
 {
+  double results = gz::math::NAN_D;
   const auto *joint = this->ReferenceInterface<JointInfo>(_id);
   const auto *identifier = std::get_if<InternalJoint>(&joint->identifier);
-  if (identifier)
-  {
+
+  if (identifier) {
```

Comment:
nit: gazebo coding style:


```suggestion
  if (identifier) 
  {
```